### PR TITLE
cli: add device selection query if more than one device connected

### PR DESCRIPTION
**Pull request by guysalt**

### CODE REVIEW

File: pymobiledevice3/cli/cli_common.py [hunk 1]
```diff
@@ -57,6 +75,22 @@ def udid(ctx, param, value):
         if '_PYMOBILEDEVICE3_COMPLETE' in os.environ:
             # prevent lockdown connection establishment when in autocomplete mode
             return
+        if value is None:
```
Comment (doronz88) [inline pymobiledevice3/cli/cli_common.py]: check instead `if value is not None` so you can simply return before and prevent the inner nesting

File: pymobiledevice3/cli/cli_common.py [hunk 2]
```diff
@@ -42,6 +45,21 @@ def wait_return():
 UDID_ENV_VAR = 'PYMOBILEDEVICE3_UDID'
 
 
+class DeviceInfo:
+
```
Comment (doronz88) [inline pymobiledevice3/cli/cli_common.py]: remove line

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: pymobiledevice3/cli/cli_common.py (hunk 1 above)
Comment (guysalt) [reply pymobiledevice3/cli/cli_common.py]: but then I need to return `LockdownClient(serial=value)` again if there is less then 2 'USB' devices connected

File: pymobiledevice3/cli/cli_common.py (hunk 2 above)
Comment (guysalt) [reply pymobiledevice3/cli/cli_common.py]: done

File: pymobiledevice3/cli/cli_common.py [hunk 3]
```diff
@@ -57,6 +75,22 @@ def udid(ctx, param, value):
...
+            devices = select_devices_by_connection_type(connection_type='USB')
+            if len(devices) > 1:
+                devices_options = []
+
+                for device in devices:
+                    lockdown_client = LockdownClient(serial=device.serial)
+                    device_info = DeviceInfo(lockdown_client)
+                    devices_options.append(device_info)
+
+                device_question = [inquirer.List("device",
```
Comment (matan1008) [inline pymobiledevice3/cli/cli_common.py]: `'`

Comment (guysalt) [reply pymobiledevice3/cli/cli_common.py]: changed `"` to `'`

File: pymobiledevice3/cli/cli_common.py (hunk 1 above)
Comment (doronz88) [reply pymobiledevice3/cli/cli_common.py]: correct, but according to the zen of python:
*"Flat is better than nested."*

In general  we prefer the code avoids nesting. If something can be checked and avoided beforehand - it should

Comment (guysalt) [reply pymobiledevice3/cli/cli_common.py]: ok, changed :)

**Review by doronz88 — APPROVED** [review]

**Pull request merged**